app/models/manageiq/providers/ibm_power_vc/cloud_manager.rb

```diff
@@ -379,12 +387,12 @@ def self.params_for_create
     }
   end
 
-  def node_endpoint
+  def endpoint
     endpoints.detect { |e| e.role == "default" }
   end
```

Comment:
You can probably just use the existing `default_endpoint` method from `ExtManagementSystem` base here if all you want is the default endpoint: https://github.com/ManageIQ/manageiq/blob/master/app/models/ext_management_system.rb#L535-L538

Reply:
You're right I'll use it directly.

Reply:
done